test/widget_tests/widgets/top_bar_tablet_test.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:io';
+
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mobile_app/src/blocs/post_bloc/post_bloc.dart';
+import 'package:mobile_app/src/services/post_service.dart';
+import 'package:mobile_app/src/services/user_services.dart';
+import 'package:mobile_app/src/widgets/custom_notification.dart';
+import 'package:mobile_app/src/widgets/icon_button_custom.dart';
+import 'package:mobile_app/src/widgets/logo.dart';
+import 'package:mobile_app/src/widgets/top_bar_tablet.dart';
+
+import 'package:mockito/mockito.dart';
```

Comment:
remember to fix for other files, I won't comment this problem again cause too many comment

Reply:
Done